selfdrive/controls/lib/planner.py

```diff
@@ -455,8 +455,8 @@ def update(self, CS, LaC, LoC, v_cruise_kph, user_distracted):
     events = []
     if self.model_dead:
       events.append(create_event('modelCommIssue', [ET.NO_ENTRY, ET.IMMEDIATE_DISABLE]))
-    if self.radar_dead or 'commIssue' in self.radar_errors:
-      events.append(create_event('radarCommIssue', [ET.NO_ENTRY, ET.IMMEDIATE_DISABLE]))
+    # if self.radar_dead or 'commIssue' in self.radar_errors:
```

Comment:
this should not be commented out. Rather, toyota/radar_interface.py should make an exception for this car and not send the error (similar to what's done for honda accord).